### ISSUE TITLE
Show closed pinned message again

### ISSUE THREAD

**Issue by bodqhrohro** [telegramdesktop/tdesktop#5526]
### Steps to reproduce
1. Open a group with a pinned message.
2. Close the message.

### Expected behaviour
Ability to open the message again. Should be in group menu, group info or somewhere else.

### Actual behaviour
I missed this ability years ago, and I still can't find it.

### Configuration
**Operating system:**
Debian GNU/Linux 10
**Version of Telegram Desktop:**
1.4
**Used theme**:
Classic

**Comment by SmartManoj** [comment]
Label

**Comment by mudrd8mz** [comment]
In fact, I would expect users can't unpin a message if the group setting does not allow them to pin messages.

**Comment by Aokromes** [comment]
> In fact, I would expect users can't unpin a message if the group setting does not allow them to pin messages.

it's not unpin, instead hide pinned message.

**Comment by mudrd8mz** [comment]
How should I describe the difference to my grand-ma?

**Comment by Aokromes** [comment]
> How should I describe the difference to my grand-ma?

easy:
unpin: remove pinned message for everyone.
hide pinned message: remove pinned message for you.

**Comment by mudrd8mz** [comment]
Right. It's probably because we wanted to use the pinned message as a permanent "forced" message that everybody must see and never hide. So I was hoping that if users can pin/unpin, they can't simply hide the message.

**Comment by AlexanderKositsyn** [comment]
Why this not done yet? I can't open pinned message if I accidentally hide it!

**Comment by bodqhrohro** [comment]
I guess it's still relevant.

*slaps the metallic butt*

Get outta here, bot!

**Comment by bodqhrohro** [comment]
I researched the current behaviour, and found out that this actual issue is still not resolved. Yeah, there already are pin status markers at the messages, yeah, there is now a confirmation dialog which makes less possible to hide some pinned message *accidentally*. But if this is done already, there still seems to be no way to see the list of messages that are pinned for others.

**Comment by d3vr** [comment]
There's no other solution than to completely remove user data directory / uninstall and reinstall ?

**Comment by Philibre2020** [comment]
Same problem here, but with the __channels__ I've subscribed to (not groups).

I use Telegram Desktop v2.6.1 on Debian GNU/Linux 10

**Comment by Philibre2020** [comment]
> I researched the current behaviour, and found out that this actual issue is still not resolved. Yeah, there already are pin status markers at the messages, yeah, there is now a confirmation dialog which makes less possible to hide some pinned message _accidentally_. But if this is done already, there still seems to be no way to see the list of messages that are pinned for others.

Indeed, I've made such a mistake, which prevents me to contact a channel's author for _months_ !
Frankly, this lack of feature is a pain in the *ss.

**Comment by Philibre2020** [comment]
> There's no other solution than to completely remove user data directory / uninstall and reinstall ?

I've tried to unsubscribe and then to subscribe again to the channel, without any effect.
I've tried with the same software but from another computer : not any effect neither.
Probably if you remove every data (and, optionally, the application), you'll get back the pinned msgs banner.

**Comment by donnyquixotic** [comment]
Is there an open PR addressing this? It's been a lingering (albeit minor) issue for years.
